$OpenBSD: patch-src_formats_pngformat_cpp,v 1.1 2019/06/25 11:54:02 cwen Exp $

Fix the build on archs where char is unsigned by default (arm,ppc)
see:
https://github.com/openbabel/openbabel/commit/e26491017efefdc6aaf73d4d49f284f4e9436244

Index: src/formats/pngformat.cpp
--- src/formats/pngformat.cpp.orig
+++ src/formats/pngformat.cpp
@@ -120,7 +120,7 @@ bool PNGFormat::ReadMolecule(OBBase* pOb, OBConversion
   if(pConv->IsFirstInput())
     count=0;
 
-  const char pngheader[] = {-119,80,78,71,13,10,26,10,0};
+  const unsigned char pngheader[] = {137,80,78,71,13,10,26,10,0};
   char readbytes[9];
   ifs.read(readbytes, 8);
   
